metacat-common/src/test/groovy/com/netflix/metacat/common/QualifiedNameSpec.groovy

```diff
@@ -98,7 +98,7 @@ class QualifiedNameSpec extends Specification {
         'c/d/t/p=1' | """{"qualifiedName": "c/d/t/p=1", "catalogName": "c", "databaseName": "d", "tableName": "t", "partitionName": "p=1"}"""
     }
 
-    def 'expect exceptions when requesting parameters that a catalog does not have'() {
+    def 'Test expect exceptions when requesting parameters that a catalog does not have'() {
```

Comment:
Why rename them all to have `Test` at front? I think the point of spock allowing you to write human readable sentences is to make it not so formal. Just curious